src/hooks/profile/use-delete-account.hook.ts

```
@@ -0,0 +1,32 @@
+        throw new Error(data.message || "Failed to delete account");
+      }
+
+      return data;
+    },
+
+    onSuccess: () => {
+      toast.success("Account deleted");
+      signOut({ callbackUrl: "/login" });
+    },
+
+    onError: (err: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)